### PR TITLE
[Feature] Change KIND to an optional way on kubernetes deployment

### PR DESCRIPTION
**Pull request by chenghuichen**
### What is the purpose of the change

Currently the kubernetes deployment scripts in linkis-dist is highly coupled wtih KIND and something else. Containerized deployment on prod environment is trending, but the current scripts can only be used for test.

### Related issues/PRs

Related issues: #4426

### Brief change log

- Decouple KIND on Kubernetes deployment

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [ ] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

**Review by peacewong — APPROVED** [review]
LGTM.

**Comment by peacewong** [comment]
ping @legendtkl @jacktao007 @AaronLinOops

File: linkis-dist/helm/scripts/install-mysql.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ MYSQL_VERSION=${MYSQL_VERSION:-5.7}
 set -e
 
 # load image
-if [ "X${KIND_LOAD_IMAGE}" == "Xtrue" ]; then
+if [ "X${USING}" == "Xtrue" ]; then
```
Comment (legendtkl) [inline linkis-dist/helm/scripts/install-mysql.sh:27]: "USING" here seems to be a typo?

File: linkis-dist/helm/scripts/install-linkis.sh [hunk 2]
```diff
@@ -21,9 +21,8 @@ WORK_DIR=`cd $(dirname $0); pwd -P`
 
 KUBE_NAMESPACE=${1:-linkis}
 HELM_RELEASE_NAME=${2:-linkis-demo}
-LOCAL_MODE=${3:-true}
-
-. ${WORK_DIR}/common.sh
+USING_KIND=${3:-false}
+LOCAL_MODE=${4:-true}
```
Comment (legendtkl) [inline linkis-dist/helm/scripts/install-linkis.sh:25]: What about keep LOCAL_MODE as the same, like

```shell
LOCAL_MODE=${3:-true}
USING_KIND=${4:-false}
```

File: linkis-dist/helm/scripts/install-linkis.sh [hunk 3]
```diff
@@ -21,9 +21,8 @@ WORK_DIR=`cd $(dirname $0); pwd -P`
 
 KUBE_NAMESPACE=${1:-linkis}
 HELM_RELEASE_NAME=${2:-linkis-demo}
-LOCAL_MODE=${3:-true}
-
-. ${WORK_DIR}/common.sh
```
Comment (legendtkl) [inline linkis-dist/helm/scripts/install-linkis.sh:26]: Deleting the shell `common.sh` scripts invoke is expected?

**Review by legendtkl — COMMENTED** [review]
Thanks for the job. @chenghuichen 

For the modification, it would be better to upload some tested result in the local environment, such as screenshot, or messages.

Thanks.

**Comment by chenghuichen** [comment]
@legendtkl Hi, I posted a screenshot of local test result below. The deployment itself looks good, except some restarts of ecm pod.

image

 The restarts of pod are caused by BML error, and not related to code change of this time.

**Pull request merged**